langs/go.go

```diff
@@ -65,6 +65,8 @@ func (h *GoLangHelper) DockerfileBuildCmds() []string {
 
 func (h *GoLangHelper) DockerfileCopyCmds() []string {
 	return []string{
+		"COPY --from=build-stage /usr/local/go/lib/time/zoneinfo.zip " +
```

Comment:
This is only a Go issue?

Reply:
So, i'm not sure, but faced with this while was developing a function with go

Reply:
we should just have it in base image we build from? string of issues over this, made some suggestions https://github.com/fnproject/dockers/pull/13 - we probably need to add for all langs (like certs)